TradeItIosTicketSDK2.xcodeproj/project.pbxproj

```diff
@@ -33,8 +33,8 @@
 		52E4AE931E4FA36C006F073A /* URLComponents+TradeIt.swift in Sources */ = {isa = PBXBuildFile; fileRef = 52E4AE911E4FA36C006F073A /* URLComponents+TradeIt.swift */; };
 		750100681F1D0FB200CC9E56 /* TradeItBrokerLogoService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 750100671F1D0FB200CC9E56 /* TradeItBrokerLogoService.swift */; };
 		750100691F1D0FB200CC9E56 /* TradeItBrokerLogoService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 750100671F1D0FB200CC9E56 /* TradeItBrokerLogoService.swift */; };
-		750171A8209B68290094B739 /* YahooNumericToggleInputTableViewCell.xib in Resources */ = {isa = PBXBuildFile; fileRef = 750171A7209B68290094B739 /* YahooNumericToggleInputTableViewCell.xib */; };
-		750171A9209B68290094B739 /* YahooNumericToggleInputTableViewCell.xib in Resources */ = {isa = PBXBuildFile; fileRef = 750171A7209B68290094B739 /* YahooNumericToggleInputTableViewCell.xib */; };
+		750171A8209B68290094B739 /* TradeItNumericToggleInputTableViewCell.xib in Resources */ = {isa = PBXBuildFile; fileRef = 750171A7209B68290094B739 /* TradeItNumericToggleInputTableViewCell.xib */; };
```

Comment:
Did you check all the Yahoo screens still work?

Reply:
Not all but most, will go over them and check.